### PR TITLE
fix and improve Utils package

### PR DESCRIPTION
**Pull request by AlbertSuarez**

### CODE REVIEW

**Comment by casassg** [comment]
Could somebody add some tests please?

**Comment by AlbertSuarez** [comment]
No worries, I'm doing it.

File: app/src/main/java/dev/blunch/blunch/utils/Utils.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import com.firebase.client.Firebase;
+
+/**
+ * Utils class
+ * @author albert
+ */
+public class Utils {
+
+    public static String generateId() {
+        return new Firebase("https://blunch.firebaseio.com/").push().getKey();
```
Comment (casassg) [inline app/src/main/java/dev/blunch/blunch/utils/Utils.java]: Why do you need to generate the ID? Why don't you just just Firebase native push method?

Comment (AlbertSuarez) [inline app/src/main/java/dev/blunch/blunch/utils/Utils.java]: I will need in the future. Trust me!

Comment (casassg) [inline app/src/main/java/dev/blunch/blunch/utils/Utils.java]: YAGNI? @quim-motger ?

Comment (AlbertSuarez) [inline app/src/main/java/dev/blunch/blunch/utils/Utils.java]: The future is this afternoon! And we will use it! I will explain tomorrow dont worry.

**Comment by casassg** [comment]
We need to discuss stuff here

**Comment by casassg** [comment]
We will need to find a better way to test it, but for the moment it just works.

* Test as an instrumentation Android test

**Pull request merged**